contributing.md

```diff
@@ -69,8 +69,9 @@ for Java. We don't have IDE support for other languages in Bazel right now.
 *  Clone Bazel's Git repository from Gerrit:
    *  `git clone https://bazel.googlesource.com/bazel`
 *  Try to build Bazel:
-   *  `cd bazel && bazel build //src:bazel`
-*  This should produce a working Bazel binary in `bazel-bin/src/bazel`.
+   *  `cd bazel && bazel build //src:bazel` (Unix)
+   *  `cd bazel && bazel build //src:bazel.exe` (Windows)
```

Comment:
Please indicate the shell that you recommend using, e.g. like so:

*  On Linux/macOS, in Bash/Terminal:
    ```
    cd bazel
    bazel build //src:bazel
    ```
*  On Windows, in the Command Prompt:
    ```
    cd bazel
    bazel --output_user_root=c:\tmp build //src:bazel.exe
    ```

Reply:
Done, thanks! Hopefully, we can get rid of the `--output_user_root` option soon with the short object file path.